requirements/common.txt

```diff
@@ -191,3 +191,7 @@ git+https://github.com/rafaelmartins/pyoembed.git@eb9901917c2a44b49e2887c077ead8
 
 # Needed to run tests in parallel
 tblib==1.3.0
+
+# Needed for oauth
+django-cors-middleware==1.3.1
+django-oauth-toolkit==0.12.0
```

Comment:
I was away due to class - lab submissions. Now I'll add the dependencies and screenshots.